docs/core/deploying/native-aot/index.md

```diff
@@ -134,6 +134,7 @@ Native AOT apps have the following limitations:
 - Implies compilation into a single file, which has known [incompatibilities](../single-file/overview.md#api-incompatibility).
 - Apps include required runtime libraries (just like [self-contained apps](../index.md#publish-self-contained), increasing their size as compared to framework-dependent apps).
 - <xref:System.Linq.Expressions> always use their interpreted form, which is slower than run-time generated compiled code.
+- Generic parameters substituted with struct type arguments will have specialized code generated for each instantiation. In CoreCLR, many instantiations are generated on-demand. In Native AOT, all instantiations are pre-generated. This can have significant impact to the disk size of the application. Generic virtual methods and generic instance methods will also have an instantiation for every implementing or overriding type.
```

Comment:
```suggestion
- Generic parameters substituted with struct type arguments will have specialized code generated for each instantiation. In dynamic runtime, instantiations are generated on-demand. In Native AOT, all instantiations are pre-generated. This can have significant impact to the disk size of the application. Generic virtual methods and generic instance methods will also have an instantiation for every implementing or overriding type.
```
We describe Native AOT as a flavor or CoreCLR. For example, later in this doc "including a stripped-down version of the coreclr runtime".

There is no single name used for non-Native AOT. We use names like "dynamic", "regular" or "general purpose".